### PR TITLE
Feature/mailchimp Martí Malek

### PR DESCRIPTION
**Pull request by martimalek**

### CODE REVIEW

File: staff/marti-malek/mailchimp/index.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            <img class="first__monkey" src=".\styles\easy-email-newsletters-banner-png8.png" alt="chimp">
+            <img class="first__title" src=".\styles\easy-email-newsletters-text.png" alt="">
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. A vel minus maiores maxime quas, facere reiciendis architecto. Ad pariatur, atque praesentium adipisci tempora similique minima provident quidem corporis molestias tenetur.</p>
+            <button class="button">Sign Up Free</button>
+        </section>
+        <section class="center">
+            <div class="container">
+                <h3>What's New</h3>
+                <section class="one">
+                    <img src=".\styles\templates.jpg" alt="">
```
Comment (Arcoders) [inline staff/marti-malek/mailchimp/index.html:38]: why you use a inverted \

Comment (martimalek) [reply staff/marti-malek/mailchimp/index.html:38]: Windows copies folder paths with this by default.

File: staff/marti-malek/mailchimp/index.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                    <h3>Golden Monkeys</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Facere minus beatae officia! Sint doloremque atque unde dicta incidunt aliquid consequuntur quae nobis quo non, modi nulla eveniet quis similique architecto.</p>
+                    <a href="">Learn more</a>
+                    <img class="arrow" src=".\styles\arrow.png" alt="arrow">
+                </section>
+            </div>     
+        </section>
+    </main>
+    <footer>
+        <section class="one">
```
Comment (lexbarba) [inline staff/marti-malek/mailchimp/index.html:73]: as a navigation bar, should be nav>ul>li>a

Comment (martimalek) [reply staff/marti-malek/mailchimp/index.html:73]: I don't think so, thx for the feedback though!

**Pull request merged**